packages/interface/src/components/file/Inspector.tsx

```diff
@@ -35,20 +35,25 @@ export const Inspector = (props: {
 	location?: LocationResource;
 	selectedFile?: FilePath;
 }) => {
-	const file_path = props.selectedFile;
-	let full_path = `${props.location?.path}/${file_path?.materialized_path}`;
+	
+	const file_path = props.selectedFile,
+	file_id = props.selectedFile?.file?.id || -1;
+	if (window.navigator.platform === 'Win32') {
+		var full_path = `${props.location?.path}\\${file_path?.materialized_path}`;
+	} else {
+		var full_path = `${props.location?.path}/${file_path?.materialized_path}`;
+	}
```

Comment:
use an array here instead, maybe:

```suggestion
	const pathSeparator = window.navigator.platform === 'Win32' ? '\\' : '/';

	full_path = [props.location?.path, file_path?.materialized_path].join(pathSeparator);
```